tuf/api/metadata.py

```diff
@@ -146,8 +152,13 @@ def from_json_file(
     # Serialization.
     def to_dict(self) -> JsonDict:
         """Returns the JSON-serializable dictionary representation of self. """
+
+        signatures_arr = []
```

Comment:
Nit: this variable name seems odd (I think arr implies array, but the data structure is a list). Why not just use `signatures`, like in `from_dict` above?